### DISCUSSION TITLE
when first creating loading bay the goods(whatever the bay is suppose to load) is not showing up until vehicle doing the second trip

### DISCUSSION THREAD

**Discussion by ozraelised** [OpenTTD/OpenTTD#10750] [Q&A] [answered]
hi all, I used to play the original game and never had this problem.
When I create, for example, a loading bay next to a diamond mine and the armoured truck enters the bay for the fist time, the bay is empty and only starts showing the diamond begs after the truck exist the loading bay. 
This is painful when you have a long route. Is there away to have the loading bay activated without the vehicle getting to the bay? 
I did a long search and couldn't find anything about this. Your help is appreciated.

Thank you,

**Comment by glx22** [comment, marked as answer]
You can type `set selectgoods false` in the console to restore the old behaviour.
The new behaviour is usually preferred because it prevents filling stations with unwanted cargo, like when you have a station in radius of a coal mine and a forest and you are only interested with coal, the wood will fill the station and drop the rating.

**Reply by ozraelised** [reply]
Thank you for the support: 

Oh, I see what you mean. Is it possible to do per bay, or is this a global setting?

**Reply by Eddi-z** [reply]
This is a global setting.

i suggest you just set the truck to "full load", then it won't leave empty on the first trip.

**Reply by ozraelised** [reply]
Both make sense - I can then change the truck to just normal
